llvm/docs/GitHub.rst

```diff
@@ -176,6 +176,25 @@ request will understand that you're rebasing just your patches, and display
 this result correctly with a note that a force push did occur.
 
 
+Pre-merge Continuous Integration (CI)
+-------------------------------------
+
+Multiple checks will be applied on a pull-request, either for linting/formatting
+or some build and tests. None of these are perfect and you will encounter
+false positive, infrastructure failures (unstable or unavailable worker), or
+you will be unlucky and based your change on a broken revision of the main branch.
+
+None of the checks are strictly mandatory: these are tools to help us build a
+better codebase and be more productive (by avoiding issues found post-merge and
+possible reverts). As a developer you're empowered to exercise your judgement
+about bypassing any of the checks when merging code.
+
+The infrastructure can print messages that make it seem like these are mandatory,
+but this is just an artifact of GitHub infrastructure and not a policy of the
+project.
+
+However, please make sure you do not force-merge any changes that have clear test failures directly linked to your changes. Our policy is still to keep the `main` branch in a good condition, and introducing failures to be fixed later violates that policy.
```

Comment:
```suggestion
However, please make sure you do not force-merge any changes that have clear test failures directly linked to your changes. Our policy is still to keep the ``main`` branch in a good condition, and introducing failures to be fixed later violates that policy.
```
Also this bit here doesn't seem to follow the 80 column limit of the rest.

Reply:
I wrapped it, thanks!